CONTRIBUTING.md

```diff
@@ -1,145 +1,125 @@
-# Contributing
+# Contributing to one-app-router
 
-The following guidelines must be followed by all contributors to this repository. Please review them carefully and do not hesitate to ask for help.
+✨ Thank you for taking the time to contribute to this project ✨
 
-### Code of Conduct
+## 📖 Table of Contents
 
-* Review and test your code before submitting a pull request.
-* Be kind and professional. Avoid assumptions; oversights happen.
-* Be clear and concise when documenting code; focus on value.
-* Don't commit commented code to the main repo (stash locally, if needed).
+* [Code of Conduct](#code-of-conduct)
+* [Developing](#developing)
+* [Submitting a new feature](#submitting-a-new-feature)
+* [Reporting bugs](#reporting-bugs)
+* [Contributing](#getting-in-contact)
+* [Coding conventions](#coding-conventions)
 
-### Git Commit Guidelines
+## Code of Conduct
+
+This project adheres to the American Express [Code of Conduct](./CODE_OF_CONDUCT.md). By contributing, you are expected to honor these guidelines.
 
-We follow precise rules for git commit message formatting. These rules make it easier to review commit logs and improve contextual understanding of code changes. This also allows us to auto-generate the CHANGELOG from commit messages.
+## Developing
 
-Each commit message consists of a **header**, **body** and **footer**.
+### Installation
 
-#### Header
+1. Fork the repository `one-app-router` to your GitHub account.
+2. Afterwards run the following commands in your terminal
 
-The header is required and must not exceed 70 characters to ensure it is well-formatted in common git tools. It has a special format that includes a *type*, *scope* and *subject*:
+    ```bash
+    $ git clone https://github.com/<your-github-username>/one-app-router
```

Comment:
not everyone uses `https` when cloning repos. Do we need to provide more than "clone the repository"

Reply:
Added a link to the tutorial for cloning a repository. I'm using this as example and provide the resources if anyone would like to use ssh.